sonar-python-plugin/src/test/java/org/sonar/plugins/python/MockSonarLintIssue.java

```diff
@@ -151,11 +158,33 @@ public boolean isQuickFixAvailable() {
 
   @Override
   public Optional<String> ruleDescriptionContextKey() {
-    throw new IllegalStateException("Not supposed to be tested");
+    return Optional.empty();
   }
 
-  protected boolean getSaved(){
-    return saved;
+  @Override
+  public List<QuickFix> quickFixes() {
+    return parent.quickFixes();
   }
 
+  @org.jetbrains.annotations.Nullable
```

Comment:
Usually we use `javax` annotations. For return types we use `@CheckForNull`

Reply:
oops, wrong import